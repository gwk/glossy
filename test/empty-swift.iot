{
  'compile': ['$PROJ/legs/__main__.py test/empty.legs -test -output $NAME.swift'],
  'cmd': './$NAME.swift',
  'args': [
    'a b c',
    'abc def',
  ],
}
