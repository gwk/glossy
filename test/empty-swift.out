
arg1: "a b c"
token: arg1:1:1-6: invalid
  a b c
  ~~~~~

arg2: "abc def"
token: arg2:1:1-8: invalid
  abc def
  ~~~~~~~
